src/config/app.yaml

```diff
@@ -1,7 +1,22 @@
-COUCH_SQS_CLIENT_CONFIG:
-  BASE_URL: http://localhost:4984
-
 NOTIFICATION:
   COUCH_DB_CLIENT_CONFIG:
     BASE_URL: http://localhost:5984
     TARGET_DATABASE: notification-webhook
+
+REPORT:
+  COUCH_DB_CLIENT_CONFIG:
+    REPORT:
+      BASE_URL: http://localhost:5984
+      TARGET_DATABASE: app
+    REPORT_CALCULATION:
+      BASE_URL: http://localhost:5984
+      TARGET_DATABASE: report-calculation
+  COUCH_SQS_CLIENT_CONFIG:
+    BASE_URL: http://localhost:4984
+    SCHEMA_DESIGN_CONFIG: /app/_design/sqlite:config
+
+REPORT_CHANGES:
+  COUCH_DB_CLIENT_CONFIG:
+    BASE_URL: http://localhost:5984
+    TARGET_DATABASE: app
```

Comment:
by definition of our platform architecture, this must be the same as REPORT_COUCH_DB. Let's reuse that config?